pages/common/bash.md

```diff
@@ -1,33 +1,33 @@
 # bash
 
-> Bourne-Again SHell, an `sh`-compatible command line interpreter.
+> Bourne-Again SHell, a Bash-compatible command line interpreter.
```

Comment:
So, Bash is Bash-compatible. I'm confused.

Reply:
Oops, I was doing a bit too much replacing in files... 😄 
```suggestion
> Bourne-Again SHell, an `sh`-compatible command line interpreter.
```